resources/Microsoft.NET.SDK.ToolInstaller/Microsoft.NET.SDK.ToolInstaller.psm1

```diff
@@ -0,0 +1,297 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+using namespace System.Collections.Generic
+
+$ErrorActionPreference = "Stop"
+$PSNativeCommandUseErrorActionPreference = $true
+Set-StrictMode -Version Latest
+
+#region Functions
+function Get-DotNetPath {
+    if ($IsWindows) {
```

Comment:
Are you only supporting default install paths for each of these? Customers can add dotnet to the path themselves or do an app-local install like we do in our own repos.

Reply:
It's still as TODO task on the class. I will try to add this option also.

Reply:
It has been added